lib/services/flowdock.ts

```diff
@@ -120,8 +120,16 @@ export class FlowdockService extends Messenger implements ServiceEmitter, Servic
 	 * @returns     Promise that resolves to the emit suitable form.
 	 */
 	public makeSpecific = (data: TransmitContext): Promise<FlowdockEmitContext> => {
-		// Build a string for the title, if appropriate.
+		// Calculate details such as title, footer and snipped message, if appropriate.
+		const lengthLimit = 8096;
```

Comment:
This might not make sense in this context, but maybe it's worth exporting this to a global constant, and also checking the env so it can be changed without a redeploy:
```typescript
const LENGTH_LIMIT = process.env.LENGTH_LIMIT || 8096

export class FlowdockService ...